lib/server/controllers/logs.controller.js

```diff
@@ -281,6 +283,40 @@ const getLogById = async (request, response, next) => {
     }
 };
 
+/**
+ * Endpoint to list all the root logs
+ *
+ * @param {Object} request the express request
+ * @param {Object} response the express response
+ * @return {void}
+ */
+const getRootLogs = async (request, response) => {
+    const value = await dtoValidator(GetAllLogsDto, request, response);
+
+    if (value) {
+        try {
+            const { query: { page: { limit = ApiConfig.pagination.limit } = {} } } = value;
+
+            // Get the logs for the current page
+            const { count, logs } = await logService.getAllRootLogs({ ...value, query: { ...value.query, page: { limit } } });
+
+            const totalPages = Math.ceil(count / limit);
+
+            await response.status(200).json({
+                data: logs,
+                meta: {
+                    page: {
+                        pageCount: totalPages,
+                        totalCount: count,
+                    },
+                },
+            });
```

Comment:
We have a function to not have to do that manually: `countedItemsToHttpView`
You can have a look and try to use it there.

Reply:
done